src/main/scala/com/lightbend/play/spring/SpringBuilder.scala

```diff
@@ -178,13 +183,46 @@ abstract class SpringBuilder[Self] protected (
     if (confClasses.nonEmpty) {
       ctx.register(confClasses: _*)
     }
-
     ctx.refresh()
     ctx.start()
 
+    println("play context started.")
+
     injector
   }
 
+  def lodBootContext(springConfig: Seq[String]) = {
+
+    val confClasses: Seq[Class[_]] = springConfig.map(className => loadClass(className))
+
+    val builder = new org.springframework.boot.builder.SpringApplicationBuilder().web(false);
+    if (confClasses.nonEmpty) {
+      println("Loading boot context with sources: " + confClasses.mkString(","))
+      builder.sources(confClasses: _*);
+
+      val rootContext: ConfigurableApplicationContext = builder.run();
+      println("Loaded boot context successfully.")
+      Some(rootContext)
+    } else {
+      None
+    }
+  }
+
+  private def getBootContext = {
+
+    configuration.getOptional[Seq[String]]("play.spring.boot.configs").map(springConfig => {
+
+      if (environment.mode == play.api.Mode.Dev && SpringBuilder.rootContext.isDefined) {
```

Comment:
It could be good to have some code comments explaining why we are checking `SpringBuilder.rootContext` here.